modules/ROOT/pages/oracle/oracle-ebs-connector.adoc

```diff
@@ -196,16 +198,16 @@ JMS, you need to supply drivers that come with your Oracle software.
 ** Host: Host of the Oracle EBS instance.
 ** Port: Port of the Oracle EBS web services.
 ** SSL: If enabled, will make calls using HTTPS instead of HTTP.
-** TrustStore file name: The custom TrustStore file for HTTPS calls (optional).
+** TrustStore file name: The custom Trust Store file for HTTPS calls (optional).
 ** TrustStore password: The password for the TrustStore (optional).
-** Disable Common Name Checking: Disables Common Name (CN) Checking on SSL certificates (optional).
+** Disable Common Name Checking: Disables common name (CN) checking on SSL certificates (optional).
 ** Responsibility Name: The responsibility name that is needed to execute the operation.
 ** Responsibility Application Name: The application short name that is needed to execute the operation.
 ** Security Group Name: The security group key of the Oracle EBS instance (optional). Default value is STANDARD.
 ** NLS Language: The NLS language of the Oracle EBS instance (optional). Default value is AMERICAN.
 ** Org ID: The organization ID of the Oracle EBS instance (optional). Default value is 204.
 +
-.. If you intent to use the connector to subscribe to Business Events or to the XML Gateway queue, you need to create a JMS Configuration. This one supports two connection types: one where you set the DB URL as a whole, and one to define it as individual parameters.
+.. To use the connector to subscribe to Business Events or to the XML Gateway queue, create a JMS configuration. This one supports two connection types: one where you set the database URL as a whole, and one to define it as individual parameters:
```

Comment:
DITTO:
```suggestion
.. To use the connector to subscribe to Business Events or to the XML Gateway queue, create a JMS configuration. 
This one supports two connection types: one where you set the database URL as a whole, and one to define it as individual parameters:
```

Reply:
I am seeing substeps and sublist items, which indicates an awful lot of information nesting.
When you have the time, try to rework topic organization so that this does not happen, perhaps by promoting topics or breaking up complicated tasks into subtasks.